requirements/AUR_CBI_MDL-0030/interfaceview.xml

```diff
@@ -0,0 +1,40 @@
+<InterfaceView asn1file="AUR_CBI_MDL-0030.asn">
+<Function name="Environment" is_type="NO" language="C" default_implementation="default" fixed_system_element="NO" required_system_element="NO" FPGA_Configurations="" startup_priority="1">
+    <Property name="Taste::coordinates" value="62700 30900 82700 38900"/>
+<Provided_Interface name="Trigger" kind="Cyclic" enable_multicast="true" layer="default" required_system_element="NO" stack_size="1024" wcet="0" priority="1">
+    <Property name="Taste::coordinates" value="70600 32600"/>
+</Provided_Interface>
+
+<Required_Interface name="ObetTime" kind="Unprotected" enable_multicast="true" layer="default" required_system_element="NO" wcet="0">
+    <Output_Parameter name="nanoseconds" type="ULongInteger" encoding="NATIVE"/>
+    <Property name="Taste::coordinates" value="62700 34700"/>
+    <Property name="Taste::InheritPI" value="true"/>
+    <Property name="Taste::Autonamed" value="true"/>
+</Required_Interface>
+
+    <Implementations>
+        <Implementation name="default" language="C"/>
+    </Implementations>
+</Function>
+
+<Function name="TimeService" is_type="NO" language="C" default_implementation="default" fixed_system_element="NO" required_system_element="NO" FPGA_Configurations="" startup_priority="1">
```

Comment:
Can we include the entire set of TimeService interfaces?

Reply:
Yes, done